Minor cleanup

bocekm opened the pull request:
Exit instead of continue if XML template not found.
Make exceptions more verbose.
Another minor cleanup:
  - code wrap on 80 chars
  - classmethod to staticmethod, where classmethod not needed
  - fixed typos
  - removed unused imports or variables

pirat89 commented on preupg/xmlgen/compose.py:
```diff
@@ -144,61 +157,71 @@ def perform_autoqa(cls, path_prefix, group_tree):
-        for f in sorted(six.iterkeys(group_tree), key=lambda tree_key: get_sorting_key_for_tree(group_tree, tree_key)):
+        for f in sorted(six.iterkeys(group_tree),
+                        key=lambda tree_key:
+                        get_sorting_key_for_tree(group_tree, tree_key)):
```
plase define the lambda function above instead of this two-lined split. Like:
```python
sort_key=lambda t_key: get_sorting_key_for_tree(group_tree, t_key)
for f in sorted(six.iterkeys(group_tree), key=sort_key)):
```

bocekm replied:
Ok, I'll change it but I'm getting PEP-8 E731 warning to not to assign lambda expression to a variable. And I kind of agree that there's a better/more comprehensive way to write the code - using def for defining function. All right, it's two lines instead of one, but who cares if it makes the code easier to comprehend.

bocekm commented:
@pirat89 see my reply for your comment. Anyway I changed the code you wanted to change :)
Also I fixed formatting of a message of one of the exceptions.
